File: selenium/src/java/org/openqa/selenium/internal/IdLookupStrategy.java
package org.openqa.selenium.internal;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.By;

public class IdLookupStrategy implements LookupStrategy {
    public WebElement find(WebDriver driver, String use) {
        return driver.findElement(By.id(use));
    }
}

File: dotnet/src/Selenium.WebDriverBackedSelenium/Internal/SeleniumEmulation/LookupStrategies/IdLookupStrategy.cs
using System;
using OpenQA.Selenium;

namespace Selenium
{
    /// <summary>
    /// Defines a strategy to look up elements based on the element ID.
    /// </summary>
    internal class IdLookupStrategy : ILookupStrategy
    {
        /// <summary>
        /// Finds an element using the element ID.
        /// </summary>
        /// <param name="driver">The <see cref="IWebDriver"/> to use in finding the element.</param>
        /// <param name="use">The locator string to use.</param>
        /// <returns>An <see cref="IWebElement"/> that matches the locator string.</returns>
        public IWebElement Find(OpenQA.Selenium.IWebDriver driver, string use)
        {
            return driver.FindElement(By.Id(use));
        }
    }
}
